project-6/main.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import os
+import pandas as pd
+from sqlalchemy import create_engine, Column, Integer, String, DateTime, Float
+from sqlalchemy.orm import sessionmaker, declarative_base
+from dotenv import load_dotenv
+from exceptions import ExtractFromParquetError, InsertError, ValidationError, NullValidation
+from datetime import datetime
+
+load_dotenv()
+CONNECTION = os.environ.get('connection')
+engine = create_engine(CONNECTION, echo=False)
+DATA = 'data/new_file.parquet'
+Session = sessionmaker(bind=engine)
+session = Session()
+
+Base = declarative_base()
+
+
+class Arts(Base):
+    __tablename__ = "arts_db"
+
+    id = Column(Integer, primary_key=True)
+    date_created = Column(DateTime)
+    title = Column(String)
+    artist = Column(String)
+    constituent_id = Column(Integer)
+    date_acquired = Column(DateTime)
+    medium = Column(String)
+    credit_line = Column(String)
+    classification = Column(String)
+    department = Column(String)
+    cataloged = Column(String)
+    height = Column(Float)
+    width = Column(Float)
+
+
+# Base.metadata.create_all(engine)
+
+
+class ArtsDb:
```

Comment:
Adding DB to every enity feels off, you should name things according to functionality, is this class a DB?